FlowCal/mef.py

```diff
@@ -631,21 +705,30 @@ def get_transform_fxn(data_beads,
         else:
             savefig = None
 
+        # Extract amplification type of clustering channels
+        clustering_amp_type = \
+            populations[0].amplification_type(clustering_channels)
+
         # If used one channel for clustering, make histogram
         if len(clustering_channels) == 1:
             plt.figure(figsize=(8,4))
-            FlowCal.plot.hist1d(populations,
-                                channel=clustering_channels[0],
-                                bins=256,
-                                alpha=0.75,
-                                savefig=savefig)
+            FlowCal.plot.hist1d(
+                populations,
+                channel=clustering_channels[0],
+                xscale='log' if clustering_amp_type[0][0] else 'logicle',
```

Comment:
Added comments about `clustering_amp_type` in 3aa9551ffd27606183fd28ac7d7122ba3c1ebdc1.